### PR TITLE
docs: add community dropdown

### PR DESCRIPTION
**Pull request by debs-obrien**
image

### CODE REVIEW

File: dotnet/docusaurus.config.js [hunk 1]
```diff
@@ -106,10 +106,35 @@ module.exports = {
...
+              to: 'community/live-streams',
+              label: 'Live Streams',
+            },
+            {
+              to: 'community/feature-videos',
+              label: 'Feature Videos',
+            },
+            {
+              to: 'community/Release-videos',
+              label: 'release Videos',
```
Comment (mxschmitt) [inline dotnet/docusaurus.config.js:135]:
```suggestion
              label: 'Release Videos',
```

File: nodejs/docusaurus.config.js [hunk 2]
```diff
@@ -107,10 +107,35 @@ module.exports = {
...
+              to: 'community/live-streams',
+              label: 'Live Streams',
+            },
+            {
+              to: 'community/feature-videos',
+              label: 'Feature Videos',
+            },
+            {
+              to: 'community/Release-videos',
+              label: 'release Videos',
```
Comment (mxschmitt) [inline nodejs/docusaurus.config.js:136]:
```suggestion
              label: 'Release Videos',
```

File: python/docusaurus.config.js [hunk 3]
```diff
@@ -106,10 +106,35 @@ module.exports = {
...
+              to: 'community/live-streams',
+              label: 'Live Streams',
+            },
+            {
+              to: 'community/feature-videos',
+              label: 'Feature Videos',
+            },
+            {
+              to: 'community/Release-videos',
+              label: 'release Videos',
```
Comment (mxschmitt) [inline python/docusaurus.config.js:135]:
```suggestion
              label: 'Release Videos',
```

**Comment by debs-obrien** [comment]
closing after team feedback to not include dropdowns

**Pull request closed without merging**